package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "asap",
+  "version": "1.0.0",
+  "description": "HueHueHue",
```

Comment:
It's a good practice to provide a meaningful description in your package.json file. This helps others understand what your project is about. [medium]